lib/reporter.js

```diff
@@ -249,13 +292,16 @@ var generateReport = function (options) {
                 suite.totalTime += feature.time
             }
 
+            suite.features = topLevelFeatures;
+            suite.features.summary = featuresSummary;
+
             return suite;
 
         });
 
         suite.totalTime = calculateDuration(suite.totalTime);
 
-        suite.features = featureOutput;
+        setupSubSuiteTemplates(suite);
```

Comment:
Its breaking the existing functionality with other formatted reports when there is a hierarchy of feature files, with the templates `bootstrap` , `foundation` or `simple`. 

To try - please move the `happy.feaure` and `unhappy.features` to respective directories, e.g. features/happy/happy.feature` and pls checkout the other templates HTML reports. 

To run the tests `npm run features`